pkg/cmd/build/depot.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/depot/depot-go/build"
+	"github.com/depot/depot-go/machine"
+	cliv1 "github.com/depot/depot-go/proto/depot/cli/v1"
+	"github.com/moby/buildkit/client"
+	buildkitClient "github.com/moby/buildkit/client"
+	"github.com/okteto/okteto/pkg/log/io"
+	"github.com/okteto/okteto/pkg/types"
+	"github.com/spf13/afero"
+)
+
+const (
+	depotTokenEnvVar   = "DEPOT_TOKEN"
+	depotProjectEnvVar = "DEPOT_PROJECT"
```

Comment:
```suggestion
	depotProjectIdEnvVar = "DEPOT_PROJECT_ID"
```

I think it's much better if we specify that it must be the project ID, otherwise it can be confusing for the user, whether they need to specify the project Name or ID.

Reply:
The "depot" official variables are `DEPOT_PROJECT_ID` and `DEPOT_TOKEN`.
O would prepend `OKTETO_` to follow other envvars nomenclature, and to make it clear these variables are for Okteto, not for depot configuration:
- `OKTETO_DEPOT_PROJECT_ID`
- `OKTETO_DEPOT_TOKEN`

Reply:
639120c2808fd060b13800d78e82742954bd027c